cloudinit/distros/dragonflybsd.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2020-2021 Gonéri Le Bouder
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import cloudinit.distros.freebsd
+
+
+class Distro(cloudinit.distros.freebsd.FreeBSD):
```

Comment:
If we follow my other comment, the base should be `cloudinit.distros.freebsd.Distro` then